src/Cli.Tests/ConfigGeneratorTests.cs

```diff
@@ -155,7 +155,8 @@ public void TestSpecialCharactersInConnectionString()
            ""runtime"": {
                 ""rest"": {
                   ""enabled"": true,
-                  ""path"": ""/api""
+                  ""path"": ""/api"",
+                  ""request-body-strict"": true
```

Comment:
The schema file - `dab.draft.schema.json` needs to be updated with the new element